workspace/utilities/lib/share-link.xsl

```diff
@@ -35,13 +35,16 @@
 		<xsl:param name="separator" select="' - '" />
 		<xsl:param name="url" />
 
+		<xsl:variable name="encoded-status" select="str:encode-uri($status, true())" />
+		<xsl:variable name="encoded-body" select="str:encode-uri($url, true())" />
+
 		<xsl:variable name="computed-url">
 			<xsl:choose>
 				<xsl:when test="string-length($status) != 0">
-					<xsl:value-of select="str:encode-uri(concat('?subject=', $status, '&amp;body=', $url), false())" />
+					<xsl:value-of select="concat('?subject=', $encoded-status, ' | NATIONAL',  '&amp;body=', $encoded-body)" />
```

Comment:
I do no think that the NATIONAL part is good ;)

Reply:
You are right. I have made another commit fixing this issue by using a separator param without mention of the project